template/src/pageComponents/buy-me-coffee/Introduction.tsx

```diff
@@ -0,0 +1,32 @@
+import Image from 'next/image';
+import UserAvatar from './Avatar.svg';
+
+export default function Introduction() {
+  // TODO: Fetch these values dynamically
+  const coffeeReceivedCount = 90;
+  const userName = '0xPudgy';
+  const aboutUser = 'I am a content creator producing videos about NFTs!';
+
+  return (
+    <section className="font-inter flex flex-col justify-start gap-8 md:flex-row md:items-center">
+      {/* Fetch user avatar dynamically */}
+      <Image alt="User Avatar" src={UserAvatar} className="h-28 w-28" />
```

Comment:
Please note: UserAvatar image is hard-coded for now. We need to fetch dynamically.

Reply:
@arsood / @Sneh1999 open question, what should be the image of the Avatar here? Is it like the person that deployed the contract or someone else?